extensions/mssql/src/localizedConstants.ts

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as nls from 'vscode-nls';
+const localize = nls.loadMessageBundle();
+
+// HDFS Constants //////////////////////////////////////////////////////////
+export const msgMissingNodeContext = localize('msgMissingNodeContext', 'Node Command called without any node passed');
```

Comment:
Is this used anywhere? I don't see it using a search. Can we fully remove this file? 😄